lib/ain-ocean/src/api/rawtx.rs

```diff
@@ -1,27 +1,148 @@
-use std::sync::Arc;
+use std::{str::FromStr, sync::Arc};
 
+use ain_macros::ocean_endpoint;
 use axum::{
+    extract::{Json, Path},
     routing::{get, post},
-    Router,
+    Extension, Router,
 };
-use defichain_rpc::{Client, RpcApi};
-use super::path::Path;
+use bitcoin::Txid;
+use defichain_rpc::{json::Bip125Replaceable, RpcApi};
+use rust_decimal::{prelude::ToPrimitive, Decimal};
 
-async fn send_rawtx() -> String {
-    "Sending raw transaction".to_string()
+use super::{response::Response, AppContext};
+use crate::{
+    error::ApiError,
+    model::{
+        default_max_fee_rate, MempoolAcceptResult, RawTransaction, RawTxDto, TransctionDetails,
+        WalletTxInfo,
+    },
+    Error, Result,
+};
+
+#[ocean_endpoint]
+async fn send_rawtx(
+    Extension(ctx): Extension<Arc<AppContext>>,
+    Json(raw_tx_dto): Json<RawTxDto>,
+) -> Result<String> {
+    println!("{:?}", raw_tx_dto.hex.clone());
+    validate(raw_tx_dto.hex.clone())?;
+    let mut max_fee = Some(default_max_fee_rate().unwrap().to_sat());
```

Comment:
should check `raw_tx_dto.max_fee` is some/none first then else default max fee rate